components/motor/gpio/setup.go

```diff
@@ -33,10 +33,10 @@ type Config struct {
 	MinPowerPct      float64        `json:"min_power_pct,omitempty"` // min power percentage to allow for this motor default is 0.0
 	MaxPowerPct      float64        `json:"max_power_pct,omitempty"` // max power percentage to allow for this motor (0.06 - 1.0)
 	PWMFreq          uint           `json:"pwm_freq,omitempty"`
-	DirectionFlip    bool           `json:"dir_flip,omitempty"`       // Flip the direction of the signal sent if there is a Dir pin
-	ControlLoop      control.Config `json:"control_config,omitempty"` // Optional control loop
-	Encoder          string         `json:"encoder,omitempty"`        // name of encoder
-	RampRate         float64        `json:"ramp_rate,omitempty"`      // how fast to ramp power to motor when using rpm control
+	DirectionFlip    bool           `json:"dir_flip,omitempty"`              // Flip the direction of the signal sent if there is a Dir pin
+	ControlLoop      control.Config `json:"control_config,omitempty"`        // Optional control loop
+	Encoder          string         `json:"encoder,omitempty"`               // name of encoder
+	RampRate         float64        `json:"ramp_rate_rpm_per_sec,omitempty"` // how fast to ramp power to motor when using rpm control
```

Comment:
the use of ramp rate is not in rpm per sec (is that just rps?) so idk how much this name really means or if it matters at all to fix right now

Reply:
to be more clear, it's in power percent. the input to `setPower` is increased or decreased by the ramp rate depending on if the current RPM is over or under the desired RPM

Reply:
I think rever this for now - we'll do a config clean later.

Reply:
I changed it to match what the config on app is so that ramp rate actually does something. before when it was just "ramp_rate" there was a mismatch between this and app and so no matter what you put as the ramp rate value it would just be the default

Reply:
so basically options are 1) revert it to `ramp_rate` which I think makes more sense, but makes it a useless field since it's `ramp_rate_rpm_per_sec` in app. 2) leave it as `ramp_rate_rpm_per_sec` which I think doesn't make sense as a name but it functions as expected. or 3) change the name of both to something that makes sense.

Reply:
we should make a ticket to fix app's config schema then - I think those are hardcoded, they don't need a migration. It was not being used anyway.

Reply:
we'll do that at the end of the epic We can talk a bit more about the correct approach.